testdata/xml-validate/invalid-data.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version='1.0' encoding='utf-8'?>
+
+<knora xmlns="https://dasch.swiss/schema"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="https://dasch.swiss/schema https://raw.githubusercontent.com/dasch-swiss/dsp-tools/main/src/dsp_tools/resources/schema/data.xsd"
+       shortcode="9999"
+       default-ontology="onto">
+
+    <permissions id="res-default">
+        <allow group="UnknownUser">V</allow>
+        <allow group="KnownUser">V</allow>
+        <allow group="ProjectMember">D</allow>
+        <allow group="ProjectAdmin">CR</allow>
+        <allow group="Creator">CR</allow>
+    </permissions>
+    <permissions id="res-restricted">
+        <allow group="ProjectMember">M</allow>
+        <allow group="ProjectAdmin">CR</allow>
+        <allow group="Creator">CR</allow>
+    </permissions>
+    <permissions id="prop-default">
+        <allow group="UnknownUser">V</allow>
+        <allow group="KnownUser">V</allow>
+        <allow group="ProjectMember">D</allow>
+        <allow group="ProjectAdmin">CR</allow>
+        <allow group="Creator">CR</allow>
+    </permissions>
+    <permissions id="prop-restricted">
+        <allow group="ProjectMember">M</allow>
+        <allow group="ProjectAdmin">CR</allow>
+        <allow group="Creator">CR</allow>
+    </permissions>
+
+
+    <!-- Cardinality Violation -->
+
+    <resource label="Max Cardinality Violation" restype=":SubClass" id="max-card-violation">
+        <text-prop name=":hasSimpleText">
+            <text encoding="utf8">Text Value One</text>
+            <text encoding="utf8">Text Value Two (this is a violation)</text>
+        </text-prop>
+    </resource>
+
+
+    <!-- Ontology Violations -->
+
+    <resource label="Link Value Target" restype=":NormalClass" id="link-value-target-wrong"/>
+
+    <resource label="Target Link Class Mismatch" restype=":NormalClass" id="link-value-wrong">
+        <text-prop name=":hasSimpleText">
+            <text encoding="utf8">The target of the link should be of the class CardOneResource</text>
```

Comment:
suggestion: how about moving the description of what is invalid to comments in the XML? this would work for non-text values as well as text values.